weatherman.py

```diff
@@ -0,0 +1,205 @@
+"""Weatherman
+This module generates yearly reports on weather data collected from files taken
+from the user provided directory.
+Report no. 1 gives the max/min temperature and max/min humidity of each year.
+Report no. 2 gives the hottest day of each year and the temperature on that day.
+"""
+
+import argparse
+import os
+import sys
+
+from collections import defaultdict
+
+
+DEFAULT_MAX = -99
+DEFAULT_MIN = 999
+DEFAULT_STR = 'DATE'
+
+
+def int_parse_wrapper(data):
+    """Used as a parser to return integer values.
```

Comment:
Should be something like 
```
    """
    This converts string to int and captures the exception
    Arguments:
        data (str): containing digits to convert
    Returns:
        (int): int value of string or None in case exceptions occur
    """
```